default_options.h

```diff
@@ -46,9 +46,6 @@ IMPORTANT: Some options will require "make clean" after changes */
  * much traffic. */
 #define DROPBEAR_SMALL_CODE 1
 
-/* Enable X11 Forwarding - server only */
-#define DROPBEAR_X11FWD 1
```

Comment:
please restore this and control it from local_options.

Reply:
? Check bottom